image/jpeg.h

```diff
@@ -105,6 +105,7 @@ class JPEGDecoder : public ImageDecoder, public Codec {
 	void setOutputColorSpace(ColorSpace outSpace) { _colorSpace = outSpace; }
 
 private:
+	using Codec::getPalette;
```

Comment:
@chkuendig I'm not sure I like this change. We are hiding a bigger problem with this

Reply:
Is the bigger problem that Image::Codec` isn't yet using `Graphics::Palette` or that we are inhereting from two superclasses that have the same function to begin with?

Not sure I feel comfortable to fix either. Happy to just remove this change from the PR though to get the other warnings resolved first.

Reply:
That we are inheriting from two superclasses that have the same function. Leave a TODO comment, to address it properly